BREEZE.rst

```diff
@@ -78,36 +78,46 @@ Docker Compose
 
 - **Permissions**: Configure to run the ``docker-compose`` command.
 
-Docker in WSL
```

Comment:
Actually since WSL 1 will still be supported. I would recommend to keep "Docker in WSL 1". There are also some notes when you want WSL 1 instead of 2. https://docs.microsoft.com/en-us/windows/wsl/compare-versions#exceptions-for-using-wsl-1-rather-than-wsl-2

Reply:
I think WSL 1 is no go in general. It does not work with Docker.

Reply:
Ah I forgot this is the documentation about Breeze - not running Airflow in general.

Then I would also remove WSL 1 from here, too. Sorry.